strawberry_django_plus/type.py

```diff
@@ -48,6 +46,18 @@
     resolve_model_nodes,
 )
 
+try:
+    from django.contrib.contenttypes.fields import GenericForeignKey, GenericRel
+except (ImportError, RuntimeError):  # pragma:nocover
+    if not TYPE_CHECKING:
+        # stubs which will never match
+        class GenericForeignKey:
+            ...
+
+        class GenericRel:
+            ...
```

Comment:
I think I commented about this in another file, the same applies here

Reply:
pyright will complain